### PR TITLE
add card and list components, add lib/database

### PR DESCRIPTION
**Pull request by Irazhap**

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { nowPlayingMovies } from "../lib/db";
```
Comment (NikRoe) [inline components/Card.js:1]: you can remove this import, you don't appear to be using the imported data here

File: components/Card.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function MovieCard({ id, image, title, year }) {
+  return (
+    <>
+      <div>
+        <Image src={image} alt={title} width={180} height={250} />
+        <p>{year}</p>
+        <h4>{title}</h4>
```
Comment (NikRoe) [inline components/Card.js:11]: better an h2 instead of an h4, for semantic HTML

File: components/Card.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { nowPlayingMovies } from "../lib/db";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function MovieCard({ id, image, title, year }) {
```
Comment (NikRoe) [inline components/Card.js:5]: you don't use the `id` and can remove it from the props.

And better to name the Component the same as the file (Card in this case)

File: components/Card.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <>
+      <div>
```
Comment (NikRoe) [inline components/Card.js:8]: do you need the `div` element here?

File: components/List.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+export default function NowPlayinglist() {
+  return (
+    <>
+      <h3>Now Playing</h3>
```
Comment (NikRoe) [inline components/List.js:15]:
```suggestion
      <h1>Now Playing</h1>
```

File: components/List.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { nowPlayingMovies } from "../lib/db";
+import MovieCard from "./Card";
+
+const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
```
Comment (NikRoe) [inline components/List.js:8]: this is the default direction, so you can remove this line of code

File: components/List.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+          {nowPlayingMovies.map((movie) => (
+            <li key={movie.id}>
+              <MovieCard
+                image={movie.image}
+                year={movie.year}
+                title={movie.title}
+              />
+            </li>
+          ))}
+        </StyledList>
```
Comment (NikRoe) [inline components/List.js:28]: you shouldn't have several list items as the child of another list item.

Better to write it like this:

```suggestion
          {nowPlayingMovies.map((movie) => (
            <StyledList key={movie.id}>
              <MovieCard
                image={movie.image}
                year={movie.year}
                title={movie.title}
              />
            </StyledList>
          ))}
```

File: pages/index.js [hunk 8]
```diff
@@ -1,7 +1,10 @@
...
+import NowPlayinglist from "@/components/List";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <NowPlayinglist />
+    </>
```
Comment (NikRoe) [inline pages/index.js:8]: you don't need fragments here

```suggestion
      <NowPlayinglist />
```

File: components/List.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+import MovieCard from "./Card";
+
+const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+export default function NowPlayinglist() {
```
Comment (NikRoe) [inline components/List.js:12]: if you choose to give your files a name, it is better to give your components the same name.
And naming it `MovieList` comes closer to what is actually does

```suggestion
export default function MoviesList() {
```

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Irazhap,

this is a very promising and great start!

I left some comments and notes. Please take a look at those and implement them in your code :) 

Keep it up 🙌

File: components/List.js (hunk 6 above)
Comment (Irazhap) [reply components/List.js:8]: should I create an extra component for styling?

Comment (NikRoe) [reply components/List.js:8]: this is already an extra component. You can keep it in the file where you use it, but you can also think about moving it someplace else and export it. Which makes a lot of sense when reusing styled components.

For the moment, it is fine to leave it where it is :)

File: components/List.js [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+      <ul>
+        <StyledUl>
+          {nowPlayingMovies.map((movie) => (
+            <StyledList key={movie.id}>
+              <Card image={movie.image} year={movie.year} title={movie.title} />
+            </StyledList>
+          ))}
+        </StyledUl>
+      </ul>
```
Comment (NikRoe) [inline components/List.js:32]: you now have a ul wrapping another ul. You only need one :) 

```suggestion
        <StyledUl>
          {nowPlayingMovies.map((movie) => (
            <StyledList key={movie.id}>
              <Card image={movie.image} year={movie.year} title={movie.title} />
            </StyledList>
          ))}
        </StyledUl>
```

**Review by NikRoe — APPROVED** [review]
Hi Irazhap,

this looks very nice and there is only one issue you need to fix before you can proceed:
You are currently wrapping an unordered list in another unordered list. Please remove the wrapping unordered list and then you are good to go!

🚀

**Pull request merged**